src/detect-engine-register.h

```diff
@@ -127,10 +127,12 @@ enum DetectKeywordId {
     DETECT_FRAME,
 
     DETECT_FLOW_AGE,
-    DETECT_FLOW_PKTS_TO_CLIENT,
+    DETECT_FLOW_PKTS,
     DETECT_FLOW_PKTS_TO_SERVER,
-    DETECT_FLOW_BYTES_TO_CLIENT,
+    DETECT_FLOW_PKTS_TO_CLIENT,
+    DETECT_FLOW_BYTES,
     DETECT_FLOW_BYTES_TO_SERVER,
+    DETECT_FLOW_BYTES_TO_CLIENT,
 
```

Comment:
> Registration table sees lesser entries

The commit message does not seem to match what is happening, we see more entries (and it is ok :-) )